src/k8s/subscriptions/PROD-CSTAR/terraform.tfvars

```
@@ -107,7 +107,7 @@ configmaps_bpdmsnotificationmanager = {
   NOTIFICATION_SERVICE_NOTIFY_WINNERS_UPDATE_STATUS_ENABLE         = "false"
   NOTIFICATION_SERVICE_SEND_WINNERS_TWICE_WEEKS_DAYS_FREQUENCY     = "15"
   NOTIFICATION_SERVICE_SEND_WINNERS_TWICE_WEEKS_SCHEDULER          = "0 00 12 * * ?"
-  NOTIFICATION_SERVICE_SEND_WINNERS_TWICE_WEEKS_START_DATE         = "2021-07-05"
+  // NOTIFICATION_SERVICE_SEND_WINNERS_TWICE_WEEKS_START_DATE         = "2021-07-05"
```

```suggestion
  NOTIFICATION_SERVICE_SEND_WINNERS_TWICE_WEEKS_START_DATE         = "2021-09-20"
```
my suggestion it's to force in prod vars this value to avoid unexpected changes in others envs.

---

I understand your point, but in my opinion replicating the definition of the same env var in two different points is a bit messy. If we need to customize it for each environment maybe the best approach is to define it _only_ in tf vars. In other words, in my opinion maps we are passing to `merge` operator shouldn't have common fields. Anyway this is a minor point, please feel free to commit your suggestion if you think it's a better approach. Thank you for the review

---

we can try this approach:
- prod -> all configs must be set
- uat,dev -> configs can be optional